jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarFiles.java

```diff
@@ -56,22 +55,17 @@ public static JibContainerBuilder toJibContainerBuilder(
       CommonContainerConfigCliOptions commonContainerConfigCliOptions,
       ConsoleLogger logger)
       throws IOException, InvalidImageReferenceException {
-
-    // Use AdoptOpenJDK image as the default base image.
-    JibContainerBuilder containerBuilder;
-    Optional<String> imageReference = commonContainerConfigCliOptions.getFrom();
-    if (imageReference.isPresent()) {
-      containerBuilder =
-          ContainerBuilders.create(
-              imageReference.get(), Collections.emptySet(), commonCliOptions, logger);
-    } else {
-      containerBuilder =
-          (processor.getJavaVersion() <= 8)
-              ? ContainerBuilders.create(
-                  "adoptopenjdk:8-jre", Collections.emptySet(), commonCliOptions, logger)
-              : ContainerBuilders.create(
-                  "adoptopenjdk:11-jre", Collections.emptySet(), commonCliOptions, logger);
-    }
+    String imageReference =
+        commonContainerConfigCliOptions
+            .getFrom()
+            .orElse(
+                processor.getJavaVersion() <= 8
+                    ? "eclipse-temurin:8-jre"
+                    : processor.getJavaVersion() <= 11
+                        ? "eclipse-temurin:11-jre"
+                        : "azul/zulu-openjdk:17-jre");
```

Comment:
Since this involves adding some logic, how about we put this in a helper method named something like, `deriveBaseImage`?